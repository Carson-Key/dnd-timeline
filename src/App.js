import React, { Component } from 'react'
import './App.css'
import Time from './containers/Time'
import { timeline } from './timeline.js'

class App extends Component {
  render() {
    return (
      <div>
        <Time timeline={timeline}/>
      </div>
    )
  }
}

export default App
