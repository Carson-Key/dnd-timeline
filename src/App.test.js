import React from 'react';
import renderer from 'react-test-renderer';
import App from './App';

describe('Renders App', () => {

  test('renders with no interact', () => {
    const component = renderer.create(
      <App />,
    );

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })

})
