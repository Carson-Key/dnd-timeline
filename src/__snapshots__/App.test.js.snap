// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders App renders with no interact 1`] = `
<div
  className="container"
>
  <br />
  <div
    className="row"
  >
    <div
      className="col"
    >
      <div
        className="jumbotron jumbotron-fluid"
      >
        <div
          className="container-fluid"
        >
          <center>
            <h1
              className="display-3"
            >
              Timeline
            </h1>
          </center>
        </div>
      </div>
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <label
        htmlFor="playerName"
      >
        What is Your Name:
      </label>
      <center>
        <div
          className="input-group"
        >
          <input
            className="form-control"
            id="playerName"
            onChange={[Function]}
            onKeyPress={[Function]}
            placeholder="John Smith"
            type="text"
          />
          <div
            className="input-group-append"
          >
            <button
              aria-label={null}
              className="btn btn-secondary"
              onClick={[Function]}
              type="button"
            >
              Enter
            </button>
          </div>
        </div>
      </center>
    </div>
  </div>
</div>
`;
